src/main/java/com/gdsc/studyjamapi/service/EventService.java

```diff
@@ -0,0 +1,12 @@
+package com.gdsc.studyjamapi.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class EventService {
+    public boolean doesEventExist(String eventId){
+        return "123".equals(eventId);
+    }
```

Comment:
You should add a TODO here to let others know that this method needs to be implemented later. Another small issue here is to put a space before the open curly brace.